### DISCUSSION TITLE
Error with BraTSPipeline command (Docker image)

### DISCUSSION THREAD

**Discussion by afamiliar** [CBICA/CaPTk#1398] [Q&A]
Hello, I'm attempting to run the BraTSPipeline binary in the Docker image available on Docker hub (latest) via the command line. The process runs smoothly until the DeepMedic skull-stripping step at which it throws an error. Please see terminal output attached.

Please let me know if you need more information.

[Bratspipeline_error_output.txt](https://github.com/CBICA/CaPTk/files/6039468/Bratspipeline_error_output.txt)

**Comment by sarthakpati** [comment]
Hi @afamiliar, 

Thank you for your interest in CaPTk! Is there a particular reason why you are using the Docker image and not the pre-packaged installer(s) for your platform (Windows, Linux (Ubuntu/CentOS) or macOS)? The reason why I ask this is because using the latter could _possibly_ give more debugging information.

Also, the DeepMedic portion is a Python binary, and we aren't very sure how it interacts with a Docker instance.

Cheers,
Sarthak
